test/common.py

```diff
@@ -821,12 +821,16 @@ def require_node(self):
     self.require_engine(nodejs)
     return nodejs
 
-  def require_node_canary(self):
+  def node_is_canary(self):
     nodejs = self.get_nodejs()
     if nodejs:
-      version = shared.get_node_version(nodejs)
-      if version >= (20, 0, 0):
-        self.require_engine(nodejs)
+      if 'canary' in nodejs:
+        return True
+    return False
```

Comment:
How about just `return nodefs and 'canary' in nodejs`?

Reply:
Done, except nodejs is a list, which I find weird since all the functions i've read so far seem to treat it as e.g. just having one version